### ISSUE TITLE
Multiplayer Window

### ISSUE THREAD

**Issue by Maxx9595** [mgba-emu/mgba#1002]
I continue to get a white blank screen when trying to load a rom in a second, multiplayer window. PLEASE HELP! I am trying to trade pokemon in fire red and leaf green

**Comment by endrift** [comment]
Dupe of #854

**Comment by Maxx9595** [comment]
wait but it doesnt say how to fix it?

**Comment by endrift** [comment]
That's because it's a bug in mGBA that hasn't been fixed yet. There are some troubleshooting steps but for the most part there's not much you can do on your own.

**Comment by Maxx9595** [comment]
Oh okay thanks!

**Comment by CyboZax** [comment]
it worked before 
and now its not

**Comment by RunisDauphin** [comment]
Does anyone know how to play 1 vs. 1 matches in Megaman Battle Network 3 (blue/black or white, US version) without speed dropping?
I tried playing this game to obtain one Giga chip that depends on this, but speed drops below 20% (although mGBA does not notice the speed drop) and is pretty much impossible both using a combo with VarSword (like B, B, Left, Down, Up) and getting a rank above 9. If you want, I can supply you a sav save file to check it. If helpful, my versions have the BN3 Plus hack by Prof. 9 (available in romhacking.net).

The computer on which I tried has these specs:
-/Processor: Intel Core i7-2600 @ 3,40 GHz.
-/RAM: 4 GB (4 GB usable).
-/Système Opératif (Operative System): Windows 10 v1803, architecture 64 bits.
-/Graphic Card: ?

I do not want to think what will happen to my laptop if I ever tried it.
